src/Columns/IColumn.h

```diff
@@ -183,6 +183,12 @@ class IColumn : public COW<IColumn>
     /// Could be used to concatenate columns.
     virtual void insertRangeFrom(const IColumn & src, size_t start, size_t length) = 0;
 
+    virtual void insertIndicesFrom(const IColumn * src, const size_t * indices_begin, const size_t length)
```

Comment:
Do you mind to replace `const size_t * ` and `size_t length` to `IColumn::Selector` to be consistent with other methods?

Reply:
Thanks fro your suggestion. I have fixed it.

Reply:
Thanks fro your suggestion. I have fixed it.